### PR TITLE
Feature data lookup lc

### PR DESCRIPTION
**Pull request by LisaChenn**

### CODE REVIEW

**Comment by LisaChenn** [comment]
approve also pls resolve conflicts ;D

File: integrated-scouting-apps/src/routes/lookup.tsx [hunk 1]
```diff
@@ -57,15 +100,29 @@ function DataLookup(props: any) {
...
+                    console.log("Firebase URL:", process.env.REACT_APP_FIREBASE_URL);
+                    console.log("event:", eventname);
+                    await handleChange(teamNum);
+                    form.resetFields(); // Optionally reset form fields after submission
                 }}
-            >
+                >
                 <Tabs defaultActiveKey="1" items={items} className='tabs' />
             </Form>
+
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/lookup.tsx:114]: remove

File: integrated-scouting-apps/src/routes/scoutingapp.tsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ function ScoutingPage(props: any) {
         <br></br>
         <Button className='mainbutton' href='/scoutingapp/picklists'>Picklists</Button>
       </div>
+      <iframe width="475" height="250" className='timer' src="https://vclock.com/embed/timer/#date=2024-02-28&title=Scouting+App+Done!&theme=1&ampm=1&sound=xylophone" frameBorder="0" allowFullScreen></iframe>
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/scoutingapp.tsx:35]: remove

**Review by nk6432 — CHANGES_REQUESTED** [review]

File: integrated-scouting-apps/src/public/stylesheets/style.css [hunk 3]
```diff
@@ -218,6 +218,49 @@ input[type="checkbox"],
 }
 
 
-.ant-tabs-tab.ant-tabs-tab-active {
-    margin-top: 50 px !important;
-}
\ No newline at end of file
+.ant-tabs-tab,.ant-tabs-tab-active {
```
Comment (nk6432) [inline integrated-scouting-apps/src/public/stylesheets/style.css:221]: add space after period

File: integrated-scouting-apps/src/routes/lookup.tsx [hunk 4]
```diff
@@ -3,34 +3,77 @@ import logo from '../public/images/logo.png';
...
-import TextArea from 'antd/es/input/TextArea';
+
+
 
 function DataLookup(props: any) {
     const [form] = Form.useForm();
-    const [color, setColor] = useState(false);
-    const canvasRef = useRef<ReactSketchCanvasRef>(null);
     useEffect(() => document.title = props.title, [props.title]);
+
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/lookup.tsx:13]: remove whitespace

File: integrated-scouting-apps/src/routes/lookup.tsx [hunk 5]
```diff
@@ -3,34 +3,77 @@ import logo from '../public/images/logo.png';
...
-
-
-    function Search() {
-        type FieldType = {
-            teamNum?: number;
+   
+   // let teamNum = '0';
+   const [teamNum, setTeamNum] = useState(0);
+  
+
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/lookup.tsx:19]: remove whitespace

File: integrated-scouting-apps/src/routes/lookup.tsx [hunk 6]
```diff
@@ -3,34 +3,77 @@ import logo from '../public/images/logo.png';
...
         };
-
-
+        try {
+          await fetch(process.env.REACT_APP_FIREBASE_URL as string, {
+            method: "POST",
+            body: JSON.stringify(body),
+            headers: {
+              "Access-Control-Request-Headers": "Content-Type, Origin",
+              "Content-Type": "application/json",
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/lookup.tsx:32]: remove headers

File: integrated-scouting-apps/src/routes/lookup.tsx [hunk 7]
```diff
@@ -3,34 +3,77 @@ import logo from '../public/images/logo.png';
...
+                className="lookupInput"
+                onChange={(value) => setTeamNum(value ?? 0)}
+                />
+            </Form.Item>
+            <Input type="submit" value="Submit" className='submitLookupButton' name='submitButton'/>
+          </div>
         );
-    }
+      }
+    // function Search() {
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/lookup.tsx:60]: remove

File: integrated-scouting-apps/src/routes/lookup.tsx [hunk 8]
```diff
@@ -57,15 +100,29 @@ function DataLookup(props: any) {
...
+                onFinish={async (values: { teamNum: any }) => {
+                    const { teamNum } = values;
+                    console.log("team Number:", teamNum);
+                    console.log("Firebase URL:", process.env.REACT_APP_FIREBASE_URL);
+                    console.log("event:", eventname);
+                    await handleChange(teamNum);
+                    form.resetFields(); // Optionally reset form fields after submission
                 }}
-            >
+                >
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/lookup.tsx:111]: remove tabs

**Review by nk6432 — CHANGES_REQUESTED** [review]
fix changes

**Pull request merged**